flow/connectors/clickhouse/normalize_query.go

```diff
@@ -193,7 +193,7 @@ func (t *NormalizeQueryGenerator) BuildQuery(ctx context.Context) (string, error
 					peerdb_clickhouse.QuoteIdentifier(dstColName),
 				)
 			}
-		case "JSON", "Nullable(JSON)":
+		case "JSON":
```

Comment:
Can be combined into:

```suggestion
		case "JSON", "Nullable(JSON)":
			stringType := strings.Replace(clickHouseType, "JSON", "String", 1)
			fmt.Fprintf(&projection,
				"JSONExtract(_peerdb_data, %s, '%s')::%s AS %s,",
				peerdb_clickhouse.QuoteLiteral(colName),
				stringType,
				clickHouseType,
				peerdb_clickhouse.QuoteIdentifier(dstColName),
			)
			if t.enablePrimaryUpdate {
				fmt.Fprintf(&projectionUpdate,
					"JSONExtract(_peerdb_match_data, %s, '%s')::%s AS %s,",
					peerdb_clickhouse.QuoteLiteral(colName),
					stringType,
					clickHouseType,
					peerdb_clickhouse.QuoteIdentifier(dstColName),
				)
			}

```